Product/Production/Adapters/General/CONNECTAdapterPDPOpenSSO/src/main/java/com/sun/identity/xacml/plugins/XSPAXACMLAuthzDecisionQueryHandler.java

```diff
@@ -210,30 +210,30 @@ public XSPAXACMLAuthzDecisionQueryHandler() {
                 LOG.warn("xspa.handleQuery():" + "DENY: based upon OPT-OUT");
             }
```

Comment:
The Log.warn statement message could be just one string. No need to concatenate here.